collections/_release_4_4/entry-highlights-jolt-physics-module.md

```diff
@@ -0,0 +1,25 @@
+---
+type: entry
+section: highlights
+rank: 5
+importance: 1
+anchor: "jolt-physics-module"
+title: "Jolt physics module"
+blockquote: <span class="highlight">Jolt</span> into action
+text: |
+  The Jolt module has been the _de facto_ physics engine for many Godot developers since its inception in late 2022, so it only made sense to promote it to an official addon for more users to find. [Jolt](https://github.com/jrouwe/JoltPhysics) itself is actually a standalone open source physics engine by [Jorrit Rouwe](https://github.com/jrouwe), Lead Game Tech Programmer at Guerrilla Games, who helped immensely with the Godot bindings.
```

Comment:
@jrouwe Are you ok with how we introduce you? Do you want us to change the phrasing?

Reply:
Can you remove `, Lead Game Tech Programmer at Guerrilla Games`? I'm not sure what my employer thinks of being mentioned and it's too much hassle to find out.

Reply:
No problem!